packages/vulkan-loader-generic/fix-hardcoded-pathes.patch

```diff
@@ -0,0 +1,25 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -149,9 +149,9 @@
+ set(LOADER_HELPER_FOLDER "Helper Targets")
+
+ if(UNIX)
+-    set(FALLBACK_CONFIG_DIRS "/etc/xdg" CACHE STRING
++    set(FALLBACK_CONFIG_DIRS "@TERMUX_PREFIX@/etc/xdg" CACHE STRING
+             "Search path to use when XDG_CONFIG_DIRS is unset or empty or the current process is SUID/SGID. Default is freedesktop compliant.")
+-    set(FALLBACK_DATA_DIRS "/usr/local/share:/usr/share" CACHE STRING
++    set(FALLBACK_DATA_DIRS "@TERMUX_PREFIX@/usr/share" CACHE STRING
```

Comment:
`@TERMUX_PREFIX@/usr` translates to /data/data/com.termux/files/usr/usr

Reply:
... I'll fix it tomorrow.